src/scripts/datadog-docs.js

```diff
@@ -839,6 +840,16 @@ function navClickEventHandler(event) {
 
     newUrl = event.target.closest('li').querySelector('a').href;
 
+    /*
+    If this is a multi code lang page we are clicking on
+    lets check if we have a lang already and load that url
+    */
+    const type = event.target.closest('li').querySelector('a').dataset.type;
+    if(type === 'multi-code-lang') {
+      const codeLang = Cookies.get('code-lang') || document.documentElement.dataset.pageCodeLang || 'python';
```

Comment:
this might never happen, but if for some reason there is no `code-lang` cookie and no `pageCodeLang` data attribute, can we be certain there will always be a page for python?

Reply:
Yea that is a good point and it doesn't feel right to do it here. Now i've tried a few different options I'm thinking the best approach might be to just make sure the nav that the user clicks on is already showing the language they are on if possible. This seems better than trying to catch this during async loading or some other area.

Reply:
@bgdeutsch i've committed a different approach, if you get time to take a look at that instead and see what you think.